.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,13 @@
 <!-- Please read CONTRIBUTING.md before submitting your pull request -->
 
 * [ ] Pull request is based on the develop branch
-* [ ] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog)
+* [ ] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
+  - Make sense to users. "Fixed a bug that prevented receiving
+    messages from other servers" instead of "Moved X method
+    from `EventStore` to `EventWorkerStore`".
+  - Use markdown where necessary, mostly for `code blocks`.
+  - Consist of a single line.
```

Comment:
this isn't *necessarily* true. https://github.com/matrix-org/synapse/blob/master/CHANGES.md#synapse-100rc1-2019-06-07 contains a multiline example (and yes, I had to go back to 1.0.0 to find one).

I'd instead change the first point to be "Be a short description of your change which makes sense to users...".

Reply:
> this isn't necessarily true. https://github.com/matrix-org/synapse/blob/master/CHANGES.md#synapse-100rc1-2019-06-07 contains a multiline example (and yes, I had to go back to 1.0.0 to find one).

Do we have a preference? I'll remove it for now.

Reply:
the preference is that normally one line is enough, but sometimes it's useful to expand a bit.

I think just saying "short description" in the first line is clear enough to be helpful but vague enough to allow some appropriate wiggle room.